example/seed-crds/10-crd-autoscaling.k8s.io_hvpa.yaml

```diff
@@ -0,0 +1,1278 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
You can add a TODO in the `hack/generate-seed-crds.sh` script to generate the HVPA CRD as well once https://github.com/gardener/hvpa-controller/issues/90 is resolved.